ui/page/livestreamOverlay/view.jsx

```diff
@@ -0,0 +1,301 @@
+// @flow
+import * as React from 'react';
+import Card from 'component/common/card';
+import Page from 'component/page';
+import ChannelSelector from 'component/channelSelector';
+import Spinner from 'component/spinner';
+import SettingsRow from 'component/settingsRow';
+import { FormField } from 'component/common/form';
+
+const DEBOUNCE_REFRESH_MS = 1000;
+
+// ****************************************************************************
+// ****************************************************************************
+
+type Props = {
+  activeChannelClaim: ChannelClaim,
+  settingsByChannelId: { [string]: PerChannelSettings },
+  fetchCreatorSettings: (channelId: string) => void,
+};
+
+export default function SettingsOverlayPage(props: Props) {
+  const { activeChannelClaim, settingsByChannelId, fetchCreatorSettings } = props;
+
+  const [lastUpdated] = React.useState(1);
+
+  // **************************************************************************
+  // **************************************************************************
+
+  // Re-sync list on first idle time; mainly to correct any invalid settings.
+  React.useEffect(() => {
+    if (lastUpdated && activeChannelClaim) {
+      const timer = setTimeout(() => {
+        fetchCreatorSettings(activeChannelClaim.claim_id);
+      }, DEBOUNCE_REFRESH_MS);
+      return () => clearTimeout(timer);
+    }
+  }, [lastUpdated, activeChannelClaim, fetchCreatorSettings]);
```

Comment:
This whole `lastUpdated` and debouncing is not needed for your case.  The effect probably just needs to listen to `activeChannelClaim.claim_id` and fetch the settings when it changes.

Also, since your component is only using the claim ID, consider using the leaner `selectActiveChannelId` instead.

Reply:
It is now required.